### PR TITLE
feat: added badges feature

### PR DESCRIPTION
**Pull request by vivekgudipudi**
image

### CODE REVIEW

File: components/badges/badges.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 0.6rem;
+    font-weight: var(--bold);
+    padding: 1px;
+    top: 6px;
+    right: 2px;
+    height: 16px;
+    width: 16px;
```
Comment (nishasen) [inline components/badges/badges.css:44]: for 16px or more use rem

**Review by nishasen — APPROVED** [review]
Suggested a small change else everything looks good

**Review by MiryalaAkshitha — APPROVED** [review]
looks good!

File: components/badges/badges.css (hunk 1 above)
Comment (vivekgudipudi) [reply components/badges/badges.css:44]: yeah changed, Thanks

**Pull request merged**